static/app/components/events/autofix/v2/autofixConfigureSeer.tsx

```diff
@@ -0,0 +1,170 @@
+                </SeerPreviewText>
+              )
+            )}
+          </Stack>
+        </SeerPreviewPanel>
+        <SeerPreviewPanel alignSelf="center">
+          <Stack gap="md" padding="md">
+            <Heading as="h3">{t('Initial Guess')}</Heading>
+            {isPending ? (
+              <Placeholder height="1rem" />
+            ) : isError ? (
+              <Stack gap="sm" direction="row">
+                <IconWarning data-test-id="error-indicator" color="gray300" size="lg" />
+                <Text>{t('Error loading initial guess')}</Text>
+              </Stack>
+            ) : (
+              data?.possibleCause && (
+                <SeerPreviewText>
+                  <MarkedText text={data?.possibleCause?.replace(/\*\*/g, '')} />
+                </SeerPreviewText>
+              )
+            )}
+          </Stack>
+        </SeerPreviewPanel>
+        <SeerPreviewPanel alignSelf="flex-end">
+          <Stack gap="md" padding="md">
+            <Heading as="h3">{t('Next Steps')}</Heading>
+            <Text>
+              {t(
+                'This is the initial analysis, but once Seer is configured you’ll be able to see a detailed breakdown of the issue’s root cause, a multi-step solution, and proposed code changes to fix it.'
+              )}
+            </Text>
+            <Stack align="start">
+              <LinkButton
+                priority="primary"
+                to={`/settings/${organization.slug}/seer/onboarding/`}
+                icon={<IconSeer />}
+              >
+                {t('Set Up Seer')}
+              </LinkButton>
+            </Stack>
+          </Stack>
+        </SeerPreviewPanel>
+      </Stack>
+    </Fragment>
+  );
+}
+
+export const SeerFeaturesPanel = styled(Panel)<{width: CSSProperties['width']}>`
+  width: ${p => p.width};
+  min-width: ${p => p.width};
+  max-width: ${p => p.width};
+`;
+
+const SeerPreviewPanel = styled(Panel)<{alignSelf: CSSProperties['alignSelf']}>`
+  align-self: ${p => p.alignSelf};
+  width: 70%;
+  min-width: 70%;
+  max-width: 70%;
+`;
+
+export const ImageContainer = styled('div')<{
+  aspectRatio?: CSSProperties['aspectRatio'];
+  height?: CSSProperties['height'];
+  maxWidth?: CSSProperties['maxWidth'];
+  minWidth?: CSSProperties['maxWidth'];
+  width?: CSSProperties['width'];
+}>`
+  display: flex;
+  ${p => p.aspectRatio && `aspect-ratio: ${p.aspectRatio}`};
+  ${p => p.height && `height: ${p.height}`};
+  ${p => p.width && `width: ${p.width}`};
+  ${p => p.minWidth && `min-width: ${p.minWidth}`};
+  ${p => p.maxWidth && `max-width: ${p.maxWidth}`};
+`;
+
+const AngledImageContainer = styled('div')`
+  position: absolute;
+  right: -50px;
+  width: 250px;
+  height: 120px;
+  transform: rotate(45deg); /* Rotates the image 45 degrees clockwise */
+`;
+
+export const Image = styled('img')<{alignSelf?: CSSProperties['alignSelf']}>`
+  align-self: ${p => p.alignSelf ?? 'center'};
+  justify-self: center;
+  width: 100%;
+`;
```

Comment:
we have an `Image` component in scraps that’s already a `styled('img')`. Could we use this here ?